CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Bug Fixes
+
+* Uses GitHub-Flavored Markdown syntax for anchors when rendering README pages.
```

Comment:
2 trailing spaces on the last line describing the change please. (from https://github.com/realm/jazzy/blob/master/CONTRIBUTING.md)